lib/install/tailwindcss.rb

```diff
@@ -16,6 +17,15 @@
   say %(        Add <%= stylesheet_link_tag "tailwind", "inter-font", "data-turbo-track": "reload" %> within the <head> tag in your custom layout.)
 end
 
+if DEVELOPMENT_ENVIRONMENT_CONFIG_PATH.exist?
```

Comment:
This will enable the "server process" only for new installs.

I opted for this solution over creating an initializer file because:
- It's one file less to manage
- Config options belong to `config/environments/*`, not initializers